casts/countryPicker/package.json

```diff
@@ -34,16 +34,17 @@
         "src/"
     ],
     "dependencies": {
+        "@tonlabs/localization": "^2.3.0",
         "@tonlabs/uicast.bars": "^2.3.0",
-        "@tonlabs/uikit.themes": "^2.3.0",
-        "@tonlabs/uikit.core": "^2.3.0",
         "@tonlabs/uikit.controls": "^2.3.0",
-        "@tonlabs/uikit.scrolls": "^2.3.0",
-        "@tonlabs/uikit.popups": "^2.3.0",
+        "@tonlabs/uikit.core": "^2.3.0",
         "@tonlabs/uikit.layout": "^2.3.0",
         "@tonlabs/uikit.media": "^2.3.0",
-        "@tonlabs/localization": "^2.3.0",
+        "@tonlabs/uikit.popups": "^2.3.0",
+        "@tonlabs/uikit.scrolls": "^2.3.0",
+        "@tonlabs/uikit.themes": "^2.3.0",
         "fuse.js": "^3.4.6",
+        "react-native-android-keyboard-adjust": "^1.2.0",
```

Comment:
This one should be a `peerDependency` (as well as devDependency), not a regular dependency!